### ISSUE TITLE
is there anyway I can get access to pro version codes? 

### ISSUE THREAD

**Issue by ShankarKakumani** [SimpleMobileTools/Simple-Gallery#2230]
I am developing an application, where I need crop, straighten, filters. 
The straightening feature is not available in the free version. 

I can pay for these pro features. I will change the design. I just need the functionality.

![134067218-882e4d7b-cdd4-4e76-b9bf-9a0b6ea4af44](https://user-images.githubusercontent.com/35686278/135761313-db1d76f8-4ed5-4937-8567-7515fe283b2e.jpg)

**Comment by tibbi** [comment]
that editor is not open source, we dont have its code either

**Comment by ShankarKakumani** [comment]
where can I find that editor, someone must be selling it right!!

any leads?

**Comment by tibbi** [comment]
they are selling the license, not the code. It is at https://github.com/imgly/pesdk-android-demo , you could have found it really easily if you actually checked our source code

**Comment by ShankarKakumani** [comment]
Oh sorry, I really didn't check the code.

I am looking for straightening from past two weeks. I can achieve straightening part alone.

But when I couldn't combine, (Straightening, Crop). Like how it's working in your pro version.


I bought the pro version from playstore and as soon as I checked the features, then I cloned the project and installed apk from the build. Then I got the normal app (without pro). I have checked for branches and other issues.

Then created a issue to ask directly.


The pricing of the SDK is too much. If it's one time, I could have bought it.

Anyway, Thanks mate.

Thanks for all the open-source projects.❤️
